ts/src/profiler.ts

```diff
@@ -97,7 +119,7 @@ function isRequestProfile(prof: any): prof is RequestProfile {
 }
 
 /**
- * @return true if response has statusCode.
+ * @return true iff response has statusCode.
  */
 // tslint:disable-next-line: no-any
 function hasHttpStatusCode(response: any):
```

Comment:
I think using `any` makes sense here, because `hasHttpStatusCode` is type-guarding to confirm response has the fields of http.ServerResponse that are used. This is necessary, because the type of response was lost since it was returned in an array of objects. 
I have slightly updated this type-guarding, though.